src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -498,7 +498,7 @@ private bool IsBetterDeletionMatch(FilterResult result1, FilterResult result2)
 
             private static int GetRecentItemIndex(ImmutableArray<string> recentItems, CompletionItem item)
             {
-                var index = recentItems.IndexOf(CompletionHelper.GetDisplayTextForMatching(item));
+                var index = recentItems.IndexOf(item.DisplayText);
```

Comment:
note: this is super weird.  effectively, this means the intellicode 'star' prefix now becomes part of the MRU...  That doesn't seem to make any sense whatsoever.